src/utils/sort.ts

```diff
@@ -27,16 +26,16 @@ export function sortByDomain(data, property, direction = 'asc', domain) {
 
 export function sortByTime(data, property, direction = 'asc') {
   return data.sort((a, b) => {
-    const aDate = moment(a[property]);
-    const bDate = moment(b[property]);
+    const aDate = a[property].getTime();
+    const bDate = b[property].getTime();
```

Comment:
Those examples are actually date objects, but the JSON representation on the demo page displays them as strings. Here's where they are generated: https://github.com/swimlane/ngx-charts/blob/master/demo/data.ts#L177

Also, we don't treat strings as dates yet, but my original intention was to do that.

There is no breaking change in this, but I'd like to hear whether you guys think we should treat strings as dates or not?